### ISSUE TITLE
Add info about gradient accumulation in logging

### ISSUE THREAD

**Issue by xyhuang** [mlcommons/logging#78]

**Comment by mwawrzos** [comment]
Indicates, after what number of steps optimizer step is called.
If submission uses no gradient accumulation, the value `1` should be reported.

Here is an example in the RNN-T reference: https://github.com/mlcommons/training/blob/651e7c47bcbd7f4708d633afa567205a826438f1/rnn_speech_recognition/pytorch/train.py#L222

**Comment by xyhuang** [comment]
done by #83
